frontend/pulse3d/src/pages/login.js

```diff
@@ -66,7 +66,7 @@ export default function Login() {
               ? router.push("/new-user")
               : router.push("/uploads"); // routes to next page
           } else {
-            res.status === 401 || res.status === 422
+            res.status === 422
```

Comment:
somehow here, we will need to notify the user if the credentials are invalid. This would be the only request that might need 401 handling still. The other's shouldn't